### ISSUE TITLE
Create a basic 3D model from already collected images

### ISSUE THREAD

**Issue by adyvoynova** [DabHands/museum-project#36]
**Describe the solution you'd like**
We should start developing the two main functionalities of the project - the image processing and the 3D model. As a start, research how to create a 3D model and try to implement a basic 3D model from the already taken images.

**Scenarios to cover**
- [x] Find the provided images and import them in the project
- [x] Research how to create a 3D model of an object, using pictures
- [x] Install VisualSfM, to create the 3D model
- [x] Install CMVS binaries to create a dense reconstruction of the model
- [x] Create a basic 3D model on your own

**Comment by adyvoynova** [comment]
In order to create the 3D model, [VisualSfM](http://ccwu.me/vsfm/index.html) needs to be installed. Using VisualSfM, we could create a 'cloud of points', which is the base level of the model. After creating the cloud of points, we can generate surfaces for it, using [MeshLab](https://www.meshlab.net/). The final part of creating a basic 3D model is to display in the browser, with the help of three.js

**Comment by adyvoynova** [comment]
There are a couple of videos, that give information about the installation of VisualSfM, its binaries and creating a basic 3D model:
- [Part 1](https://www.youtube.com/watch?v=Z61_6ehH3nQ&ab_channel=KyleYounkin)
- [Part 2](https://www.youtube.com/watch?v=5nu21nh64NY&ab_channel=KyleYounkin)

There are also three more videos that demonstrate how to use VisualSfM, together with MeshLab:
- [Video 1](https://www.youtube.com/watch?v=ZPfM52oo_S0&ab_channel=Tim)
- [Video 2](https://www.youtube.com/watch?v=ZRTEMKS3Sw0&ab_channel=DavePSB)
- [Video 3](https://www.youtube.com/watch?v=Nsl0fUtc2ug&ab_channel=gtoonstra)

**Comment by adyvoynova** [comment]
I managed to create a model of the 22 images that Terry sent us of the sculpute. The final result is attached below. I created the model using the VisualSfM GUI. First, multiple images should be imported, then the missing matches should be computed. The third step is to create a sparse model, and then bundles should be added. The last step is to create a dense reconstruction. This is the most time consuming step. For 22 images, it took around 18 minutes to complete. To view the final model in the VisualSfM GUI, I pressed the TAB button on the keyboard.

![Screenshot 2022-10-21 174537](https://user-images.githubusercontent.com/107405811/197249924-ad3cd35e-e157-448e-877f-c484d12f167d.png)
